src/lang-promql/client/prometheus.ts

```diff
@@ -188,7 +188,7 @@ export class HTTPPrometheusClient implements PrometheusClient {
       'match[]': labelMatchersToString(metricName, matchers, labelName),
     });
     // See https://prometheus.io/docs/prometheus/latest/querying/api/#finding-series-by-label-matchers
-    return this.fetchAPI<Map<string, string>[]>(`${seriesEndpoint}?${params}`).catch((error) => {
+    return this.fetchAPI<Map<string, string>[]>(`${seriesEndpoint}?${params}`, {method: 'post'}).catch((error) => {
```

Comment:
in case it is really usefull to move to the POST method (looking forward to the answer to my above question :)), then you also have to put the parameter in the body and to set the content type like it is explained in the documentation you pointed.

For the moment, the only effect of this PR is the HTTP method changed from GET to POST. But the query parameters are still there and the content type didn't change.

![image](https://user-images.githubusercontent.com/4548045/115985099-83408880-a5aa-11eb-8f1f-9d49de3531bf.png)